### PR TITLE
iep 404 moved 600 spec to public, added insolvency details in overview

### PR DESCRIPTION
**Pull request by gshivach**
added insolvency details to overview
added mock responses details to overview

### CODE REVIEW

File: assets/sections/manipulate-company-data-api-filing/templates/guides/overview.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ API Filing initially exposes two API services:
 
 - `Transactions API`. This API provides the generic model that encapsulates all other API Filing services. A transaction is best thought of as an envelope for one or more data changes or forms.
 - `Registered Office Address (ROA) API`. This API provides the specific ROA data change functionality. An ROA data change can be added to a transaction before the transaction is submitted to Companies House.
+- `Insolvency API`. This API provides the specific Insolvency data change functionality - for example to open an insolveny case and appoint an insolvency practitioner (equivalent to the paper '600' form). An Insolvency data change can be added to a transaction before the transaction is submitted to Companies House.
```
Comment (DTaylor-CH) [inline assets/sections/manipulate-company-data-api-filing/templates/guides/overview.md:12]: typo ('insolveny')

Comment (DTaylor-CH) [reply assets/sections/manipulate-company-data-api-filing/templates/guides/overview.md:12]: Insolvency capitalisation consistency - pick one

File: assets/sections/manipulate-company-data-api-filing/templates/guides/overview.md [hunk 2]
```diff
@@ -22,12 +23,15 @@ All API Filing software will need to sign their users in to Companies House with
...
+| Service/API | Required scope(s) | Additional information |
+|---|---|---|
+| Transactions (pre-requisite for all other APIs) | `https://identity.company-information.service.gov.uk/user/profile.read` | |
+| Registered Office Address (ROA) | `https://api.company-information.service.gov.uk/company/{company_number}/registered-office-address.update` | Note that the `{company_number}` placeholder must be filled in with an actual company number e/g/ for company `00000001` this scope would be `https://api.company-information.service.gov.uk/company/00000001/registered-office-address.update`. |
+| Insolvency | `https://api.company-information.service.gov.uk/company/*/insolvency.write-full` | Due to the sensitivity of insolvency endpoints, this scope can only be granted to a client (i.e. your software) if your client_id is registered with Companies House as a recognised insolvency software - regardless of whether or not the user granting the scope is a registered insolvency practitioner. |
+
 
 Combining multiple scopes is done by space separating them as one string so the requested scope for API Filing an ROA data change for company `00000001` should request the scope:
```
Comment (DTaylor-CH) [inline assets/sections/manipulate-company-data-api-filing/templates/guides/overview.md:36]: What scope? Something should follow the colon

**Review by DTaylor-CH — APPROVED** [review]

File: assets/sections/manipulate-company-data-api-filing/templates/guides/overview.md [hunk 3]
```diff
@@ -22,12 +23,15 @@ All API Filing software will need to sign their users in to Companies House with
 
 - `https://identity.company-information.service.gov.uk/user/profile.read` is required for API Filing and grants permission to read the users profile data.
 
-Depending on the filing functionality the list of scopes may change. However, for the ROA change, the following scope will also be required:
+Depending on the filing functionality the list of scopes may change.
```
[the remark this answers is not in the document]
Comment (gshivach) [reply assets/sections/manipulate-company-data-api-filing/templates/guides/overview.md:31]: added now

**Pull request merged**